docs/public/using-devfile-odo.dev.push.path-attribute.adoc

```diff
@@ -0,0 +1,29 @@
+# Using the odo.dev.push.path related attributes
+
+`odo` uses the `odo.dev.push.path` related attribute from the devfile's run commands to push only the specified files and folders to the component. The format of the attribute is `"odo.dev.push.path:<local_relative_path>": "<remote_relative_path>"`
+
+```yaml
+commands:
+  - id: dev-run
+    attributes:
+      "dev.odo.push.path:target/quarkus-app": "remote-target/quarkus-app"
```

Comment:
You mean wherever is being pushed, name of the remote folder will be `remote-target`. I am getting confused because the remote target mentioned https://github.com/openshift/odo/pull/4681/files#diff-690405b0010238bcf5d1edadd2272c958de0ff87da1d35d75ce9ad3f2fca50a0R29 is `target/quarkus-app`
Could you please sync both the places 🙂

Reply:
Fixed